store/src/java/com/zimbra/cs/account/ldap/AutoProvision.java

```diff
@@ -432,7 +433,8 @@ protected void sendNotifMessage(Account acct, String password)
         String toAddr = acct.getName();
 
         try {
-            SMTPMessage out = new SMTPMessage(JMSession.getSmtpSession());
+            Domain domain = Provisioning.getInstance().getDomain(acct);
```

Comment:
Can we make this into a single utility method...

Reply:
can you please provide me an example for this as reference?

Reply:
Below 2 lines are repeated in multiple classes
```
Domain domain = Provisioning.getInstance().getDomain(owner);
SMTPMessage report = new SMTPMessage(JMSession.getSmtpSession(domain));
```
We should create a utility method which accept account as arg and return `SMTPMessage` object as output.

Reply:
ok, any suggestions on which class file I can add this method?

Reply:
AccountUtil I think, because we are fetching details from Account object.

Reply:
Done above changes.